src/Nri/Ui/Tooltip/V1.elm

```diff
@@ -0,0 +1,635 @@
+    viewTooltip_ AuxillaryDescription
+
+
+{-| Supplementary information triggered by a "?" icon
+
+A toggle tip is always triggered by a click.
+
+-}
+toggleTip :
+    { onTrigger : Bool -> msg
+    , isOpen : Bool
+    , label : String
+    , extraButtonAttrs : List (Attribute msg)
+    }
+    -> Tooltip msg
+    -> Html msg
+toggleTip { isOpen, onTrigger, extraButtonAttrs, label } tooltip_ =
+    Nri.Ui.styled Html.div
+        "Nri-Ui-Tooltip-V1-ToggleTip"
+        tooltipContainerStyles
+        []
+        [ Html.button
+            ([ Widget.label label
+             , css
+                ([ Css.width (Css.px 20)
+                 , Css.height (Css.px 20)
+                 , Css.color Colors.azure
+                 ]
+                    ++ buttonStyleOverrides
+                )
+             , Events.onClick (onTrigger True)
+             , Events.onBlur (onTrigger False)
+             ]
+                ++ extraButtonAttrs
+            )
+            [ iconHelp ]
+        , viewIf (\_ -> viewCloseTooltipOverlay (onTrigger False)) isOpen
+        , Html.span
+            [ -- This adds aria-live polite & also aria-live atomic, so our screen readers are alerted when content appears
+              Role.status
+            ]
+            [ -- Popout is rendered after the overlay, to allow client code to give it
+              -- priority when clicking by setting its position
+              viewIf (\_ -> viewTooltip Nothing tooltip_) isOpen
+            ]
+        ]
+
+
+{-| Made with <https://levelteams.com/svg-to-elm>
+-}
+iconHelp : Svg msg
+iconHelp =
+    svg [ width "20px", height "20px", viewBox "0 0 25 25" ]
+        [ g [ stroke "none", strokeWidth "1", fill "none", fillRule "evenodd" ]
+            [ circle [ stroke "#146AFF", strokeWidth "2", cx "12.5", cy "12.5", r "11.5" ] []
+            , Svg.path [ d "M12.6825,6.6275 C13.3866702,6.6275 14.0095806,6.74395717 14.55125,6.976875 C15.0929194,7.20979283 15.5154151,7.53749789 15.81875,7.96 C16.1220849,8.38250211 16.27375,8.86458063 16.27375,9.40625 C16.27375,9.98041954 16.1329181,10.470623 15.85125,10.876875 C15.5695819,11.283127 15.1579194,11.7408308 14.61625,12.25 C14.1937479,12.6508353 13.8768761,12.9866653 13.665625,13.2575 C13.4543739,13.5283347 13.3216669,13.8262484 13.2675,14.15125 L13.18625,14.6875 L11.74,14.6875 L11.74,13.875 C11.74,13.3116639 11.8402073,12.8458352 12.040625,12.4775 C12.2410427,12.1091648 12.5362481,11.6975023 12.92625,11.2425 C13.2079181,10.9174984 13.419166,10.6385428 13.56,10.405625 C13.700834,10.1727072 13.77125,9.91541808 13.77125,9.63375 C13.77125,9.30874838 13.6602094,9.0595842 13.438125,8.88625 C13.2160406,8.7129158 12.8991687,8.62625 12.4875,8.62625 C11.7074961,8.62625 10.9437537,8.85916434 10.19625,9.325 L10.19625,7.29375 C10.9112536,6.84958111 11.7399953,6.6275 12.6825,6.6275 Z M11.17125,18.34375 L11.17125,15.7275 L13.82,15.7275 L13.82,18.34375 L11.17125,18.34375 Z", fill "#146AFF" ]
+                []
+            ]
+        ]
```

Comment:
☁️ Check in with Tessa re. how to do SVGs these days.

Reply:
checked it out, I think this is ok. There's a  new type for going between modules but since this is just used right here I think it's fine